pkg/network/protocols/http/ebpf_gotls.go

```diff
@@ -396,13 +396,16 @@ func (p *GoTLSProgram) registerProcess(binID binaryID, pid pid, mTime syscall.Ti
 }
 
 func (p *GoTLSProgram) unregisterProcess(pid pid) {
-	p.lock.Lock()
-	defer p.lock.Unlock()
-
+	p.lock.RLock()
```

Comment:
what about javatls?

Reply:
we only subscribe on exec event on javatls